hedera-node/hedera-addressbook-service-impl/build.gradle.kts

```diff
@@ -30,6 +32,7 @@ mainModuleInfo { annotationProcessor("dagger.compiler") }
 testModuleInfo {
     requires("com.hedera.node.app.service.addressbook.impl")
     requires("com.hedera.node.app.service.mono")
+    requires("com.hedera.node.app.service.mono.test.fixtures")
```

Comment:
We should not have dependencies on anything in mono going forward.

Reply:
removed thanks